window_title/window_title.lua

```diff
@@ -0,0 +1,59 @@
+local account = require('account')
+local ffi = require('ffi')
+
+ffi.cdef[[
+  bool SetWindowTextW(void* hWnd, wchar_t const* title);
+  void* GetActiveWindow();
+  ]]
+  
+local user32 = ffi.load('user32')
+
+local wide = function(str)
```

Comment:
We should provide a mechanism to do this, so I would mark this as temporary. Could be made part of #43.